### ISSUE TITLE
Inconsistent behavior with subscribe() and publish()

### ISSUE THREAD

**Issue by scottbradybuckii** [uNetworking/uWebSockets.js#564]

**Comment by hst-m** [comment]
Yes can confirm ws.publish() only works for the 1st ws.publish() called per event loop

**Comment by hst-m** [comment]
Seems to be for a specific case: ws.publish() works for a specific socket until it publishes to a topic it does not belong to, then all following ws.publish() to topics it does belong to incorrectly send to itself. To fix the problem for now you can use app.publish() instead of ws.publish() if the ws does not belong to the topic you are trying to use ws.publish() with
